src/blueprints/ember-addon/__addonLocation__/unpublished-development-types/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // These will *not* be published as part of your addon, so be careful that your published code does not rely on them!
 <% if (options.packages.addon.hasGlint) { %>
 import '@glint/environment-ember-loose';
+// import '@glint/environment-ember-template-imports';
```

Comment:
https://github.com/embroider-build/addon-blueprint/pull/218

At the moment, the codemod doesn't check if `@glint/environment-ember-template-imports` (optional) is installed. Rather than omitting the line, I opted to comment the line.